core-bundle/assets/controllers/tooltips-controller.js

```diff
@@ -2,9 +2,11 @@ import {Controller} from "@hotwired/stimulus";
 
 export default class TooltipsController extends Controller {
     static defaultOptionsMap = {
+        'a img[alt]': {x: -9, y: 30},
         'p.tl_tip': {x: 0, y: 23, useContent: true},
         '#home[title]': {x: 6, y: 42},
         '#tmenu a[title]': {x: 0, y: 42},
+        'a.link[title]': {x: -9, y: 27},
```

Comment:
There is a BC layer in place that adds the `data-contao-tooltips-target` attribute to elements that should receive a tooltip (see static `afterLoad` method in this class). Currently, the keys of the `defaultOptionsMap` also double as a source for this BC layer. 

Ideally, new elements should directly use the data attribute instead of relying on the BC layer.  As long as our tooltip code needs those x/y positions, we still need the map though. What do you think about…

- adding an array `backwardsCompatiblitySelectors` that contain the current keys (i.e. not adding the new one there),
- using that array instead of `targetSelectors` in `afterLoad`
- and adding the data attribute to the rendered HTML similar to #7841?